apm-sniffer/apm-sdk-plugin/postgresql-8.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/postgresql/util/ParameterUtil.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.skywalking.apm.plugin.jdbc.postgresql.util;
+
+import com.google.common.base.Joiner;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author aderm
+ */
+public class ParameterUtil {
+
+    public static String format(Object[] params, int limitCnt, int limitLength) {
+        return format(Arrays.asList(params), limitCnt, limitLength);
+    }
+
+    public static String format(List<Object> params, int limitCnt, int limitLength) {
+        boolean overLoad = false;
+        int actualCnt = params.size();
+        if (params.size() > limitCnt) {
+            overLoad = true;
+            actualCnt = limitCnt;
+        }
+
+        String paramStr = Joiner.on(",").join(params.subList(0, actualCnt));
```

Comment:
Introduce a third-party dependency (guava) just for one method is not worthy, and again, we **MUST NOT** use third-party dependency unless we shad them

Reply:
ok